Helper/CustomerSerializer.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Metrilo\Analytics\Helper;
+
+class CustomerSerializer extends \Magento\Framework\App\Helper\AbstractHelper
+{
+    public function __construct(
+        \Magento\Newsletter\Model\Subscriber $subscriberModel,
+        \Magento\Framework\App\RequestInterface $request
```

Comment:
What would you need the request for??

Reply:
This is redundant dependency injection. Removed.